osu.Game/Rulesets/Mods/ModJudgementChallenge.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Rulesets.Judgements;
+using osu.Framework.Bindables;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Configuration;
+using osu.Game.Graphics.UserInterface;
+using osu.Framework.Localisation;
+using osu.Game.Overlays.Settings;
+
+namespace osu.Game.Rulesets.Mods
+{
+    public abstract class ModJudgementChallenge : ModChallenge
+    {
+        public override string Name => "Judgement Challenge";
+        public override string Acronym => "JC";
+        public override IconUsage? Icon => FontAwesome.Solid.Gavel;
+        public override string Description => "Fail the beatmap if your judgements go above specified values.";
+
+        [SettingSource("Maximum non-best judgements", "Maximum number of non-best judgements before fail (disregards judgement type).", 1, SettingControlType = typeof(SettingsSlider<int, JudgementMaxSlider>))]
+        public BindableNumber<int> MaxNonBest { get; } = new BindableInt
+        {
+            Default = 26,
+            Value = 26,
+            MinValue = 0,
+            MaxValue = 26
```

Comment:
Why are these bounds hardcoded? What if the map has less than 26 hitobjects? What if you want to set more?